modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/core/schema/DataDefinition.es.js

```diff
@@ -24,4 +29,16 @@ export class DataDefinitionSchema extends Schema {
 	get availableLanguageIds() {
 		return this[SYMBOL_RAW].availableLanguageIds;
 	}
+
+	get dataDefinitionFields() {
+		return this[SYMBOL_RAW].dataDefinition.dataDefinitionFields;
+	}
```

Comment:
For more information https://github.com/liferay-forms/liferay-portal/pull/176#discussion_r610042909

Maybe I'll send another commit tomorrow morning to end this.

Reply:
Done https://github.com/liferay-forms/liferay-portal/pull/179/commits/de038c40d0ded28aef55444777d46cf8701fd8fc

See the commit message for more information.

Reply:
cc @miltonmc and @alinedoleron